src/emulator/auth/server.ts

```diff
@@ -461,10 +476,14 @@ function toExegesisController(
         // See: https://cloud.google.com/identity-platform/docs/reference/rest/v1/accounts/signUp
         targetProjectId = ctx.user;
       }
-      if (ctx.params.path.tenantId || ctx.requestBody?.tenantId) {
-        throw new NotImplementedError("Multi-tenancy is unimplemented.");
+      if (ctx.params.path.tenantId && ctx.requestBody?.tenantId) {
+        assert(
+          ctx.params.path.tenantId === ctx.requestBody?.tenantId,
+          "Tenant IDs in path and request body do not match"
```

Comment:
Needs a test (and also I believe production has a different error code for this)